### PR TITLE
Qitian (Jason) Homework

### PR DESCRIPTION
**Pull request by onjas-6**
I have experience in Markdown, so for a large portion, it goes smoothly. However, I had difficulty in issuing a fork and a pull request, it's partly because of the newbie-unfrieldliness of Github. However, with help from Deb, I successfully completed the task.

### CODE REVIEW

**Review by deblnia — COMMENTED** [review]
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      X        |            |
| **Coding strategy**         |           |     X         |            |
| **Presentation**            |           |       X       |            |
| **Achievement, creativity** |           |    X          |            |
| **Ease of access**          |           |      X        |            |

Remarks:
* Successfully used at least 4 different elements of markdown syntax (good inclusion of an image, and linking using markdown). 
* Clean repo -- no extraneous files. 
* Regarding Github workflow in the future: try to be more specific in your pull request reflection and be sure to commit as often as you'd save an essay.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
-# hw01
-Repository for homework 01
+# Qitian Hu hw01
+* Hi! I am **Qitian** or **Jason**. A first year in the *college*, potentially majoring in ***philosophy and econ***. This my github homepage [Qitian's Github Page](https://www.google.com "Qitian's homepage") . 
+* I am relatively familar with **Markdown and github**, and the hardest part is actually to *fork the repository*. With the help from Deb, I am able to do so.
+
+![Screen Shot 2019-04-05 at 1.45.20 PM](my_image.png)
```
Comment (deblnia) [inline README.md]: For some reason, I don't see this alt-text rendering on the image. You could play around with [different ways of rendering](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images) if you wanted to.

**Review by feng-liang — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    x       |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |       x    |              |            |
| **Achievement, creativity** |      x     |              |            |
| **Ease of access**          |        x   |              |            |

Hello Jason,
Good job on the assignment. I found all the required component, and the layout of the page is concise. Good choice on bolding your name, since this is an assignment on introducing yourself and your name is the most important information. Maybe try to include a caption for your picture as well.

**Comment by yanhao89** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      x        |            |
| **Coding strategy**         |           |     x         |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |    x       |              |            |

Remarks:
* Very economical introduction, with no superfluous details.
* I like the picture that you presented of yourself in bed.

**Pull request closed without merging**